com.chartboost.mediation/Runtime/AdFormats/Banner/ChartboostMediationBannerViewAndroid.cs

```diff
@@ -50,7 +50,11 @@ public ChartboostMediationBannerViewAndroid(AndroidJavaObject bannerAd) : base(n
 
         public override BidInfo WinningBidInfo
         {
-            get => _bannerAd.Get<AndroidJavaObject>("winningBidInfo").MapToWinningBidInfo();
+            get
+            {
+                var winningBidInfo = _bannerAd.Get<AndroidJavaObject>("winningBidInfo");
+                return winningBidInfo?.MapToWinningBidInfo() ?? new BidInfo();
```

Comment:
🤔 I don't think we may want to return a new BidInfo(). If it is not included in our callback, most likely it was lost or something happened at that time.

Reply:
🤔 It's a non nullable struct so `new BidInfo()` is just a default case. Does returning a `Null`sound better ? The default case imo is better since it would avoid NRE when this is called before load request is made

Reply:
Mostly cautious that `BidInfo()` here won't return with any info that could mislead users when winningBidInfo is not found. Perhaps, we could return an empty map of some sorts. Does BidInfo() return an empty object?

Reply:
yeah that's an empty object